spec/specification/consumer_spec.rb

```diff
@@ -463,25 +463,37 @@ module Pod
 
       it 'returns the on demand resources specified as a string wrapped in an array' do
         @spec.on_demand_resources = { 'Maps' => 'MapView/Map/Resources/*.png' }
-        @consumer.on_demand_resources.should == { 'Maps' => ['MapView/Map/Resources/*.png'] }
```

Comment:
Should the deleted line also be supported with a default category?

Reply:
let me check

Reply:
it actually is. consumer now will always normalize and return a hash with :path and :category. I think the test ensures that if the user doesnt use the hash DSL then consumer would do the right thing.